stable/socat-tunneller/templates/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1beta2
```

Comment:
Can you please use the `apps/v1` for the `apiVersion`  as it has been available since Kubernetes 1.9.